src/index.js

```diff
@@ -619,7 +619,9 @@ export function app(props) {
       if (typeof obj[0] === "function") {
         dispatch(obj[0](state, obj[1], props))
       } else {
-        obj[1][0](obj[1][1], dispatch, setState(obj[0]))
+        obj.slice(1).map(function(fx) {
+          fx[0](fx[1], dispatch)
+        }, setState(obj[0]))
```

Comment:
It may be premature optimization, but I think slice is expensive.
https://jsperf.com/slice-1-vs-for-loop-vs-foreach

Reply:
This is a size tradeoff I think we can afford as I don't think you'd need to dispatch more than a few effects in parallel at any given time. We can definitely revisit this in the future; it isn't saving us a lot of bytes anyway.

Thank you, @diontools. 💯